ansible/roles/ml-analytics-service/defaults/main.yml

```diff
@@ -10,7 +10,7 @@ ml_analytics_survey_app_name: "{{ ml_survey_app_name | default('sunbirdsurvey')
 ml_analytics_integrated_app_name: "{{ ml_integrated_app_name | default('sunbird') }}"
 ml_analytics_integrated_portal: "{{ ml_integrated_portal | default('dev.sunbird.portal') }}"
 ml_analytics_survey_service: "http://{{private_ingressgateway_ip}}/ml-survey"
-ml_analytics_evidence_base_url: "https://{{ sunbird_private_storage_account_name }}.blob.core.windows.net/telemetry-data-store/"
+ml_analytics_evidence_base_url: "https://{{ sunbird_public_storage_account_name }}.blob.core.windows.net/samiksha/"
```

Comment:
Can we remove this hardcoding?

Reply:
@raghupathiguduri

Reply:
Can you review now again @keshavprasadms